ddtrace/contrib/aiohttp/patch.py

```diff
@@ -1,4 +1,15 @@
+from ddtrace.internal.utils.deprecations import DDTraceDeprecationWarning
+from ddtrace.vendor.debtcollector import deprecate
+
 from ..internal.aiohttp.patch import *  # noqa: F401,F403
 
 
-# TODO: deprecate and remove this module
+def __getattr__(name):
+    deprecate(
+        ("%s.%s is deprecated" % (__name__, name)),
+        category=DDTraceDeprecationWarning,
+    )
+
+    if name in globals():
+        return globals()[name]
+    raise AttributeError("%s has no attribute %s", __name__, name)
```

Comment:
Shouldn't we avoid raising errors basically always? Log instead?

Reply:
Here we are resecting the expected behavior of python modules. If a user accesses an attribute that doesn't exist (ex: ddtrace.contrib.aiohttp.munirsucks) then an attribute error should be raised. For all existing attributes a deprecation warning is logged.